source/grid/grid_out.cc

```diff
@@ -3303,11 +3303,9 @@ namespace internal
                       const unsigned int offset=face_no*n_points;
                       // we don't need to transform the vertices: they are
                       // already in physical space.
-                      line_points.push_back(face->vertex(0));
-                      for (unsigned int i=1; i<n_points - 1; ++i)
+                      for (unsigned int i=0; i<n_points; ++i)
```

Comment:
Ha, neat. That makes the code even nicer.

But you need to update the comment above. Probably same in the duplicated code below.